Source/Core/VideoCommon/TextureConverterShaderGen.h

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <string>
```

Comment:
I'm fairly certain `<string>` and `<utility>` don't need to be included here in the header